src/common/VirtualTerminal.cpp

```diff
@@ -210,7 +210,9 @@ namespace SDDM {
 
                 // Clear VT
                 static const char *clearEscapeSequence = "\33[H\33[2J";
-                write(vtFd, clearEscapeSequence, sizeof(clearEscapeSequence));
+                if (write(vtFd, clearEscapeSequence, sizeof(clearEscapeSequence)) == -1) {
```

Comment:
Technically checking for success requires `== sizeof(clearEscapeSequence)`, but then errno isn't set and that's even more annoying. So IMO this is fine

Reply:
You're right, I wasn't sure if it is worth the effort.
I can change it if you prefer :)

Reply:
No need!